### PR TITLE
feat(tutorial): proxy-protocol-v2 ingress-nginx configuration

### PR DESCRIPTION
**Pull request by Grraahaam**
### Your checklist for this pull request

- [x] Check that the commit messages match our requested structure.
- [x] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Tutorial to setup the `ingress-nginx` to accept `proxy-protocol-v2` communication

Further details : https://github.com/scaleway/docs-content/issues/1231

### CODE REVIEW

File: tutorials/proxy-protocol-v2-load-balancer/index.mdx [hunk 1]
```diff
@@ -209,6 +209,59 @@ The Proxy Protocol support in Apache is still in an early stage. It is currently
     apache2ctl restart
     ```
 
+## Configuring Proxy protocol for `ingress-nginx`
+
+1. You'll need to add the two annotations below to your ingress controller `service` :
```
Comment (RoRoJ) [inline tutorials/proxy-protocol-v2-load-balancer/index.mdx:214]:
```suggestion
1. Add the two annotations below to your ingress controller `service` :
```

File: tutorials/proxy-protocol-v2-load-balancer/index.mdx [hunk 2]
```diff
@@ -209,6 +209,59 @@ The Proxy Protocol support in Apache is still in an early stage. It is currently
...
+    - [`service.beta.kubernetes.io/scw-loadbalancer-proxy-protocol-v2: "true"`](https://www.scaleway.com/en/docs/containers/kubernetes/api-cli/using-load-balancer-annotations/#service.beta.kubernetes.ioscw-loadbalancer-proxy-protocol-v2)
+    - [`service.beta.kubernetes.io/scw-loadbalancer-use-hostname: "true"`](https://www.scaleway.com/en/docs/containers/kubernetes/api-cli/using-load-balancer-annotations/#service.beta.kubernetes.ioscw-loadbalancer-use-hostname)
+   
+    ```bash
+    kubectl patch -n ingress-nginx services ingress-nginx-controller \
+    -p '{"metadata":{"annotations":{"service.beta.kubernetes.io/scw-loadbalancer-proxy-protocol-v2": "true", "service.beta.kubernetes.io/scw-loadbalancer-use-hostname": "true"}}}'
+    ```
+
+2. And add the below configuration to the ingress controller `configmap` :
```
Comment (RoRoJ) [inline tutorials/proxy-protocol-v2-load-balancer/index.mdx:223]:
```suggestion
2. Add the below configuration to the ingress controller `configmap` :
```
```suggestion
2. And add the below configuration to the ingress controller `configmap` :
```

File: tutorials/proxy-protocol-v2-load-balancer/index.mdx [hunk 3]
```diff
@@ -209,6 +209,59 @@ The Proxy Protocol support in Apache is still in an early stage. It is currently
...
+    - [`use-proxy-protocol: "true"`](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/#use-proxy-protocol)
+    
+    ```bash
+    kubectl patch -n ingress-nginx configmaps ingress-nginx-controller \
+    -p '{"data":{"use-forwarded-headers": "true", "compute-full-forwarded-for": "true", "use-proxy-protocol": "true"}}'
+    ```
+
+### Helm
+
+If you're using the [`ingress-nginx`](https://kubernetes.github.io/ingress-nginx/deploy/) Helm chart, here's a custom values file that will automatically create a load balancer resource and configure it to use `proxy-protocol-v2`, and add the required ingress controller configuration to support this new communication protocol :
```
Comment (RoRoJ) [inline tutorials/proxy-protocol-v2-load-balancer/index.mdx:235]:
```suggestion
If you're using the [`ingress-nginx`](https://kubernetes.github.io/ingress-nginx/deploy/) Helm chart, find below a custom values file that will automatically create a load balancer resource and configure it to use `proxy-protocol-v2`, and add the required ingress controller configuration to support this new communication protocol :
```

File: tutorials/proxy-protocol-v2-load-balancer/index.mdx [hunk 4]
```diff
@@ -209,6 +209,59 @@ The Proxy Protocol support in Apache is still in an early stage. It is currently
...
+        annotations:
+        service.beta.kubernetes.io/scw-loadbalancer-proxy-protocol-v2: "true"
+        service.beta.kubernetes.io/scw-loadbalancer-use-hostname: "true"
+    config:
+        use-forwarded-headers: "true"
+        compute-full-forwarded-for: "true"
+        use-proxy-protocol: "true"
+    ```
+
+2. To apply the above on a new or an existing `ingress-nginx` installation :
```
Comment (RoRoJ) [inline tutorials/proxy-protocol-v2-load-balancer/index.mdx:254]:
```suggestion
2. Apply the above, on a new or an existing `ingress-nginx` installation :
```

**Review by RoRoJ — COMMENTED** [review]
Thanks for your tutorial! I've just left some suggestions, mostly because one of our style guidelines is that every step should start with an action verb :-)

**Review by bene2k1 — APPROVED** [review]

**Comment by jtherin** [comment]
I will add a subtitle `### Deployed through kubectl` to avoid users using kubectl patch on ingress deployed through helm.

also
```
<Message type="important">
  Once ProxyProtocol is enabled the Apache virtual host will no longer accept direct connections.
</Message>```
is not placed correctly.

File: tutorials/proxy-protocol-v2-load-balancer/index.mdx [hunk 5]
```diff
@@ -212,3 +212,58 @@ The Proxy Protocol support in Apache is still in an early stage. It is currently
 <Message type="important">
   Once ProxyProtocol is enabled the Apache virtual host will no longer accept direct connections.
 </Message>
+
+## Configuring Proxy protocol for `ingress-nginx`
+
+### Deployed through `kubectl`
+
+1. Add the two annotations below to your ingress controller `service` :
+    - [`service.beta.kubernetes.io/scw-loadbalancer-proxy-protocol-v2: "true"`](https://www.scaleway.com/en/docs/containers/kubernetes/api-cli/using-load-balancer-annotations/#service.beta.kubernetes.ioscw-loadbalancer-proxy-protocol-v2)
```
Comment (ldecarvalho-doc) [inline tutorials/proxy-protocol-v2-load-balancer/index.mdx:221]:
```suggestion
    - [`service.beta.kubernetes.io/scw-loadbalancer-proxy-protocol-v2: "true"`](/containers/kubernetes/api-cli/using-load-balancer-annotations/#service.beta.kubernetes.ioscw-loadbalancer-proxy-protocol-v2)
```

File: tutorials/proxy-protocol-v2-load-balancer/index.mdx [hunk 6]
```diff
@@ -212,3 +212,58 @@ The Proxy Protocol support in Apache is still in an early stage. It is currently
...
+    - [`service.beta.kubernetes.io/scw-loadbalancer-use-hostname: "true"`](https://www.scaleway.com/en/docs/containers/kubernetes/api-cli/using-load-balancer-annotations/#service.beta.kubernetes.ioscw-loadbalancer-use-hostname)
```
Comment (ldecarvalho-doc) [inline tutorials/proxy-protocol-v2-load-balancer/index.mdx:222]:
```suggestion
    - [`service.beta.kubernetes.io/scw-loadbalancer-use-hostname: "true"`](/containers/kubernetes/api-cli/using-load-balancer-annotations/#service.beta.kubernetes.ioscw-loadbalancer-use-hostname)
```

File: tutorials/proxy-protocol-v2-load-balancer/index.mdx [hunk 7]
```diff
@@ -212,3 +212,58 @@ The Proxy Protocol support in Apache is still in an early stage. It is currently
...
+   
+    ```bash
+    kubectl patch -n ingress-nginx services ingress-nginx-controller \
+    -p '{"metadata":{"annotations":{"service.beta.kubernetes.io/scw-loadbalancer-proxy-protocol-v2": "true", "service.beta.kubernetes.io/scw-loadbalancer-use-hostname": "true"}}}'
+    ```
+
+2. Add the below configuration to the ingress controller `configmap` :
```
Comment (ldecarvalho-doc) [inline tutorials/proxy-protocol-v2-load-balancer/index.mdx:229]:
```suggestion
2. Add the configuration below to the ingress controller `configmap`:
```

File: tutorials/proxy-protocol-v2-load-balancer/index.mdx [hunk 8]
```diff
@@ -212,3 +212,58 @@ The Proxy Protocol support in Apache is still in an early stage. It is currently
...
+    - [`use-proxy-protocol: "true"`](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/#use-proxy-protocol)
+    
+    ```bash
+    kubectl patch -n ingress-nginx configmaps ingress-nginx-controller \
+    -p '{"data":{"use-forwarded-headers": "true", "compute-full-forwarded-for": "true", "use-proxy-protocol": "true"}}'
+    ```
+
+### Deployed through `helm`
+
+If you're using the [`ingress-nginx`](https://kubernetes.github.io/ingress-nginx/deploy/) Helm chart, find below a custom values file that will automatically create a load balancer resource and configure it to use `proxy-protocol-v2`, and add the required ingress controller configuration to support this new communication protocol :
```
Comment (ldecarvalho-doc) [inline tutorials/proxy-protocol-v2-load-balancer/index.mdx:241]:
```suggestion
If you're using the [`ingress-nginx`](https://kubernetes.github.io/ingress-nginx/deploy/) Helm chart, find below a custom values file that will automatically create a load balancer resource and configure it to use `proxy-protocol-v2`, and add the required ingress controller configuration to support this new communication protocol:
```

File: tutorials/proxy-protocol-v2-load-balancer/index.mdx [hunk 9]
```diff
@@ -212,3 +212,58 @@ The Proxy Protocol support in Apache is still in an early stage. It is currently
...
+
+1. Create the below file (e.g. `ingress-nginx-scw.yaml`)
```
Comment (ldecarvalho-doc) [inline tutorials/proxy-protocol-v2-load-balancer/index.mdx:243]:
```suggestion
1. Create the file below (e.g. `ingress-nginx-scw.yaml`) using a text editor.
```

File: tutorials/proxy-protocol-v2-load-balancer/index.mdx [hunk 10]
```diff
@@ -212,3 +212,58 @@ The Proxy Protocol support in Apache is still in an early stage. It is currently
...
+        annotations:
+        service.beta.kubernetes.io/scw-loadbalancer-proxy-protocol-v2: "true"
+        service.beta.kubernetes.io/scw-loadbalancer-use-hostname: "true"
+    config:
+        use-forwarded-headers: "true"
+        compute-full-forwarded-for: "true"
+        use-proxy-protocol: "true"
+    ```
+
+2. Apply the above, on a new or an existing `ingress-nginx` installation :
```
Comment (ldecarvalho-doc) [inline tutorials/proxy-protocol-v2-load-balancer/index.mdx:260]:
```suggestion
2. Apply the above, on a new or an existing `ingress-nginx` installation:
```

File: tutorials/proxy-protocol-v2-load-balancer/index.mdx [hunk 11]
```diff
@@ -212,3 +212,58 @@ The Proxy Protocol support in Apache is still in an early stage. It is currently
 <Message type="important">
   Once ProxyProtocol is enabled the Apache virtual host will no longer accept direct connections.
 </Message>
+
+## Configuring Proxy protocol for `ingress-nginx`
+
+### Deployed through `kubectl`
+
+1. Add the two annotations below to your ingress controller `service` :
```
Comment (ldecarvalho-doc) [inline tutorials/proxy-protocol-v2-load-balancer/index.mdx:220]:
```suggestion
1. Add the two annotations below to your ingress controller `service`:
```

**Review by ldecarvalho-doc — APPROVED** [review]

**Pull request merged**